django/contrib/postgres/operations.py

```diff
@@ -12,6 +12,9 @@ def state_forwards(self, app_label, state):
         pass
 
     def database_forwards(self, app_label, schema_editor, from_state, to_state):
+        engine = schema_editor.connection.settings_dict['ENGINE']
+        if not ('postgresl' in engine or 'postgis' in engine):
```

Comment:
apart from the missing `q` in `postgresql`: I'm -1 on this change. If I have a custom engine inheriting from the built-in PG engine this change forces me to name it appropriately. Can't we inspect the actual engine class and see the vendor?